include/jwt-cpp/jwt.h

```diff
@@ -1093,7 +1088,12 @@ namespace jwt {
 			 */
 			std::string sign(const std::string& data, std::error_code& ec) const {
 				ec.clear();
-				std::unique_ptr<EVP_MD_CTX, decltype(&EVP_MD_CTX_free)> ctx(EVP_MD_CTX_create(), EVP_MD_CTX_free);
+#ifdef JWT_OPENSSL_1_0_0
+				std::unique_ptr<EVP_MD_CTX, decltype(&EVP_MD_CTX_destroy)> ctx(EVP_MD_CTX_create(),
```

Comment:
Maybe I am wrong, but isn't there a `, EVP_MD_CTX_destroy);` missing at the end ?

Reply:
it's on the next line =D https://github.com/Thalhammer/jwt-cpp/pull/155/files#diff-be2369ece29c0945fc04ffe13a39767b7841fc2fabd3df209a26b3bb381a2fb2R1093

Reply:
Dang why is it so far to the right ? ^^
Yeah its fine then.